rest.yaml

```
@@ -249,12 +249,927 @@ paths:
+          type: "string"
+          format: "date"
+        endDate: 
+          type: "string"
+          format: "date"
+        registrationStart: 
+          type: "string"
+          format: "date"
+        registrationEnd: 
+          type: "string"
+          format: "date"
+        association: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)